packages/contracts/waffle.js

```diff
@@ -11,11 +11,11 @@ var waffleConfig = {
 };
 
 var selectSolc = () => {
-  if (process.env.CI || process.env.NATIVE_SOLC == "true") {
+  if (process.env.NATIVE_SOLC == "true") {
```

Comment:
Why did you remove the `CI` env variable?

Reply:
IIRC because the selector in master never does native solc, even on a dev's local laptop. Will fix that separately